PukingPredator/Assets/Scripts/PlayerCamera.cs

```diff
@@ -25,6 +25,5 @@ void LateUpdate()
     void Rotate()
     {
         finalOffset = Quaternion.AngleAxis(Input.GetAxis("Mouse X") * sensitivity, Vector3.up) * finalOffset;

```

Comment:
I don't think PlayerCamera is being used anymore, we can delete it in a different PR